src/views/transaction/transaction.view.jsx

```diff
@@ -156,16 +156,16 @@ function Transaction ({
                         </Row>
                         )
                       : <></>}
-                    {!transactionTask.data.fromHezEthereumAddress && transactionTask.data.fromBjj
+                    {!transactionTask.data.fromHezEthereumAddress && (transactionTask.data.fromBjj || transactionTask.data.fromBJJ)
```

Comment:
I meant when hez ethereum address is === the internal one like in the link i sent you. because there will always be an hez ethereum address, but in those cases, we should stil show the bjj